packages/backend/src/workers/provider/LlamaCppPython.ts

```diff
@@ -179,8 +179,12 @@ export class LlamaCppPython extends InferenceProvider {
       }
     }
 
+    // add the link to our openAPI instance using the instance as the host
+    const aiLabPort = this.configurationRegistry.getExtensionConfiguration().apiPort;
+    // add in the URL the port of the inference server
+    const aiLabDocsLink = `http://localhost:${aiLabPort}/api-docs/${config.port}`;
     // adding labels to inference server
-    labels['docs'] = `http://localhost:${config.port}/docs`;
+    labels['docs'] = aiLabDocsLink;
```

Comment:
why not inline?

Reply:
maybe this should be in a dedicated function, like `getDocumentionUrl(inferencePort: number)` or something like that